ibis/backends/flink/__init__.py

```diff
@@ -315,30 +343,34 @@ def create_table(
         import pyarrow_hotfix  # noqa: F401
 
         import ibis.expr.types as ir
+        from ibis.backends.flink.datatypes import FlinkRowSchema
 
         if obj is None and schema is None:
             raise exc.IbisError("The schema or obj parameter is required")
 
-        if overwrite:
-            self.drop_table(name=name, catalog=catalog, database=database, force=True)
+        # in-memory data is created as views in `pyflink`
+        elif obj is not None:
+            if isinstance(obj, pa.Table):
+                obj = obj.to_pandas()
+            if isinstance(obj, pd.DataFrame):
+                table = self._table_env.from_pandas(
+                    obj, FlinkRowSchema.from_ibis(schema)
+                )
```

Comment:
The docstring says that only one of `obj` and `schema` can be provided, but it looks like here it's not only accepted but required.

You can use `PandasSchema.to_ibis` and pass in a list of `(name, dtype)` tuples to generate a compatible Ibis schema from the input.  (PyArrow also has a similar option if you want to generate the schema there for pyarrow inputs)

Reply:
Ahh, I see the `None` handling in the `FlinkRowSchema` below now -- in that case, you can either make the change here or update the docstring to note that a schema can optionally be specified alongside a dataframe or pyarrow table

Reply:
updated the docstring.